packages/serve-static-middleware/src/base/utils.ts

```diff
@@ -0,0 +1,28 @@
+import type { ServeStaticConfig, PathConfig } from './types';
+import type { ServeStaticOptions } from 'serve-static';
+import { relative, isAbsolute, join } from 'path';
+
+/**
+ * Resolves the serve static options from a configuration object.
+ *
+ * @param options configuration object
+ * @returns resolved serve static options
+ */
+export const resolveServeStaticOptions = (options: ServeStaticConfig | PathConfig): ServeStaticOptions => {
```

Comment:
Could you please add to the function description why we are doing the filtering

Reply:
Sure, done with https://github.com/SAP/open-ux-tools/pull/1673/commits/1532eb52f836078b3adeaaab0faeaafa0884e907